client/src/js/app/app.js

```diff
@@ -0,0 +1,34 @@
+angular
+  .module('toDoList', ['ui.router',
+                       'ui.bootstrap',
+                       'ngResource']);
+
+angular.module('toDoList').config(routeConf);
+
+function routeConf($stateProvider, $urlRouterProvider) {
+  //
+  // For any unmatched url, redirect to /state1
+  $urlRouterProvider.otherwise("/login");
+  //
+  // Now set up the states
+  $stateProvider
+    .state('login', {
+      url: '/login',
+      templateUrl: 'login.html',
+      controller:  'LoginController as Ctrl'
```

Comment:
it doesn't make sense to give the same names for controllers in `controllerAs` syntax